lib/form/html.js

```diff
@@ -107,35 +106,18 @@ const addRequired = (html, input, index) => {
         return;
       }
 
-      // regex to get first class attributes from the captured label
-      const regexClass = new RegExp(`.*(${stringClass})`);
-
-      // regex to get classes out of `class`
-      const regexClasses = new RegExp('class=["\']([\\w\\s\\W]+?)["\']');
-
       // regex to search for `for`
       const reFor = new RegExp(`(${stringFor})`);
-
-      // first, check if `class` exists in label
-      if (Array.isArray(label[1].match(regexClass))) {
-        const cls = label[1].match(regexClass);
-        const classAttr = cls[1].match(regexClasses);
-        const classes = classAttr[1].split(/\s+/).concat([level]);
-        const newClass = `class="${classes.join(' ')}"`;
-        render = render.replace(label[0], label[0].replace(classAttr[0], newClass));
-      }
-      else {
-        // else, we add class after the `for` attribute
-        render = render.replace(reFor, `for="${inp.id}" class="${level}"`);
-      }
+      render = render.replace(reFor, `for="${inp.id}"`);
```

Comment:
So, just checking, all this was removed because we no longer put a className on `label` correct?